KsApi/Service.swift

```diff
@@ -306,6 +302,14 @@ public struct Service: ServiceType {
     return requestPagination(paginationUrl)
   }
 
+  public func fetchProjectCreatorDetails(query: NonEmptySet<Query>) -> SignalProducer<ProjectCreatorDetailsEnvelope, GraphError> {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/kickstarter/ios-oss/blob/a8f032cddfba991aba476d145bc2cbcf057daea7/KsApi/Service.swift#L305"></span>Line should be 110 characters or less: currently 130 characters<br />
<code>line_length</code> <code>Service.swift:305</code></td>
    </tr>
  </tbody>
</table>